apps/arena/priv/config.json

```diff
@@ -1533,5 +1559,6 @@
       "vertices": [],
       "amount_of_power_ups": 1
     }
-  ]
+  ],
+  "traps": []
```

Comment:
What's this key for?

Reply:
In case we want to add default traps, we can get them from the config

Reply:
nice